components/vf-social-links/vf-social-links.scss

```diff
@@ -0,0 +1,120 @@
+// vf-social-links
+
+// **Thinking about deleting this file?**
+// If your component needs no CSS/Sass, we still recommend leaving the
+// scss files in place. As this is primarily a CSS framework, it is better to
+// leave the empty files so you know a file wasn't accidently omitted.
+// If you don't have any Sass, you can trim this block down to:
+// "This page was intentionally left blank"
+
+@import 'package.variables.scss';
+// Debug information from component's `package.json`:
+// ---
+/*!
+ * Component: #{map-get($componentInfo, 'name')}
+ * Version: #{map-get($componentInfo, 'version')}
+ * Location: #{map-get($componentInfo, 'location')}
+ */
+
+@import 'vf-social-links.variables.scss';
+
+
+.vf-social-links__heading {
+  @include set-type(text-heading--4);
+
+  color: set-ui-color(vf-ui-color--black);
+  text-transform: capitalize;
```

Comment:
EMBL's (undocumented) style is only for sentence caps, which I don't think there is a `text-transform` for.

Reply:
I'll change this in the actual text, remove it from the CSS and hope any other team uses it follows policy.

I guess we could do something like `p:first-letter { text-transform: capitalize; }` - but at source is best here then